src/backend/marsha/core/views.py

```diff
@@ -101,6 +102,10 @@ def get_context_data(self):
             VideoSerializer(video).data if video else None
         )
 
+        context["cloudfront_domain"] = settings.CLOUDFRONT_DOMAIN
+        context["environment"] = settings.CONFIGURATION
+        context["authorized_environment"] = ["production", "preproduction", "staging"]
```

Comment:
```suggestion
        context["authorized_environments"] = ["production", "preproduction", "staging"]
```

Reply:
preproduction should be called preprod like in matsuo no?

Reply:
This is not the value used in `settings.CONFIGURATION` ? in `settings.py` the class name is `PreProduction`